prime-router/metadata/hl7_mapping/ORU_R01/RADx_MARS/RADx_MARS-base.yml

```diff
@@ -7,27 +7,60 @@ constants:
   patientAgeAoeLoinc: '"35659-2"'
```

Comment:
Changes here were intended to preserve the original behavior for RADx MARS, even though many of the formerly overridden elements were removed or edited in the base schema.